### PR TITLE
chore: Update prerelease/v4 with latest master

### PR DESCRIPTION
**Pull request by anicholls**
Also included some fixes and additions to bring things up to par in [35e0c77](https://github.com/Workday/canvas-kit/pull/688/commits/35e0c773dabd981624e3ad97cbe2f63cf5321de2)

### CODE REVIEW

**Comment by mannycarrera4** [comment]
So far my commit matches my changes 👍

**Comment by jamesfan** [comment]
My changes look good, too 👍.

@anicholls Should I approve the new Chromatic baselines for Select as well?

**Comment by lychyi** [comment]
@anicholls My changes from #651 look good here.

File: modules/badge/react/index.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export {default as CountBadge, CountBadgeProps} from './lib/CountBadge';
```
Comment (alanbsmith) [inline modules/badge/react/index.ts:1]: I don't think I want the default export from badge to be `CountBadge`.

Comment (alanbsmith) [inline modules/badge/react/index.ts:1]:
```suggestion
export {CountBadge, CountBadgeProps} from './lib/CountBadge';
```

Comment (alanbsmith) [reply modules/badge/react/index.ts:1]: nvmd. This isn't an issue.

**Pull request merged**